docs/integrations/sources/auth0.md

```diff
@@ -55,7 +56,8 @@ The connector is restricted by Auth0 [rate limits](https://auth0.com/docs/troubl
 
 | Version | Date       | Pull Request                                             | Subject                                                                        |
 |:--------|:-----------|:---------------------------------------------------------|:-------------------------------------------------------------------------------|
-| 0.3.0  | 2023-06-20 | TBD | Add Organizations, OrganizationMembers, OrganizationMemberRoles streams |
-| 0.2.0  | 2023-05-23 | 26445 | Add Clients stream |
-| 0.1.0  | 2022-10-21 | TBD | Add Auth0 and Users stream |
+|  1.0.0  | 2023-08-03 | [28972](https://github.com/airbytehq/airbyte/pull/28972) | Migrate to Low-Code CDK                                                        |
+|  0.3.0  | 2023-06-20 | TBD                                                      | Add Organizations, OrganizationMembers, OrganizationMemberRoles streams |
+|  0.2.0  | 2023-05-23 | [26445](https://github.com/airbytehq/airbyte/pull/26445) | Add Clients stream                                                             |
+|  0.1.0  | 2022-10-21 | TBD                                                      | Add Auth0 and Users stream                                                     |
```

Comment:
See comments about versioning above, also would it be too much trouble to update the TBDs here?